extensions/VisualEditor/wikia/VisualEditor.i18n.php

```diff
@@ -1851,15 +1882,15 @@
 	'wikia-visualeditor-dialog-wikiamediainsert-upload-error-size' => 'Ups! Ten obraz jest zbyt duży. $1MB - to maksymalny, dopuszczalny rozmiar pliku!',
 	'wikia-visualeditor-dialog-wikiamediainsert-upload-error-filetype' => 'Ups! Wspieramy jedynie $1',
 	'wikia-visualeditor-dialog-wikiamediainsert-upload-error' => 'Błąd podczas przesyłania pliku',
-	'wikia-visualeditor-dialog-wikiamediainsert-video-search-input-placeholder' => 'Szukaj lub wprowadź adres URL',
+	'wikia-visualeditor-dialog-wikiamediainsert-video-search-input-placeholder' => 'Szukaj filmu lub wklej zewnętrzny URL',
 	'wikia-visualeditor-dialog-wikiamediainsert-preview-alert' => 'Podgląd dostępny wkrótce!',
-	'wikia-visualeditor-dialog-wikiamediainsert-policy-message' => 'Proszę trzymać się zasad przesyłania obrazów {{SITENAME}} oraz [http://pl.wikia.com/Zasady_Użytkowania Zasad Użytkowania Fandomu] podczas dodawania nowych filmów i przesyłania nowych obrazów.',
+	'wikia-visualeditor-dialog-wikiamediainsert-policy-message' => 'Proszę trzymać się zasad przesyłania obrazów {{GRAMMAR:genitive|{{SITENAME}}}} oraz [http://pl.wikia.com/Zasady_Użytkowania Zasad Użytkowania Fandomu] podczas dodawania nowych filmów i przesyłania nowych obrazów.',
```

Comment:
@idradm Please make sure this translation works correctly. Recently we had an issue with combining parser function with magic words..